### ISSUE TITLE
[Incorrect behavior][3.0.0-beta.1.1]: config.json fails to blacklist unsharing of voices

### ISSUE THREAD

**Issue by ThatNintendoNerd** [Raytwo/ARCropolis#182]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### System Firmware Version

13.2.0

### Atmosphere Version

1.2.5

### Smash Version

13.0.1

### ARCropolis Version

3.0.0-beta.1.1

### Version channel

Beta

### SD card formatting

FAT32

### Can this incorrect behavior be consistently reproduced?

Yes

### Expected behavior

```
{
  "unshare-blacklist": [
    "sound/bank/fighter_voice/vc_mewtwo_c00.nus3audio"
  ]
}
```
Using this `config.json`, the expected end result is to hear the modified voice clips over every costume. In the provided modpack, Mewtwo's up-taunt has been replaced with an annoying half-second tone. From my testing:
- The file as-is and no `config.json` allows the mod to work properly, albeit only on c00.
- The file duplicated for all alternate costumes and no `config.json` allows the mod to work properly, but with the additional bloat I'm trying to avoid.
- Attempting to reshare the voice files using the `config.json` from above makes the game not acknowledge the mod files, and so vanilla voices are used instead for all costumes. This behavior also occurs with the following:
  - Not using a costume indicator (i.e. `vc_mewtwo.nus3audio`)
  - Using a non-c00 costume indicator. 
  - With or without using a `.nus3bank` file.

For some characters like the [Octoling](https://gamebanana.com/mods/306740) mod over Inkling, attempting to reshare voices using c04 and c07 has caused Inkling's voice to go mute altogether. This did not occur with Mewtwo.

### Configuration

```
version = '3.0.0-beta.1.1'
debug = false
auto_update = true
beta_updates = true
no_web_menus = true
region = 'us_en'

[paths]
arc = 'rom:/arc'
umm = 'sd:/ultimate/mods'
extra_paths = []

[logger]
logger_level = 'Warn'
log_to_file = false
```

### Minimal modpack to observe the issue

https://cdn.discordapp.com/attachments/676583505010098216/920171677176954910/sound_mewtwo.7z

**Comment by blu-dev** [comment]
Thank you TNN, I will take a look when I get a moment

**Comment by ThatNintendoNerd** [comment]
Resolved as of ARCropolis 3.1.0. May have been resolved at an earlier date.
